realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -151,44 +151,6 @@ public void basic_multipleReferencesFromParentList() {
         assertEquals(parent, child.getListParents().last());
     }
 
-    // A listener registered on the backlinked object should be called when a commit adds a backlink
-    @Test
-    @RunTestInLooperThread
-    public void notification_onCommitModelObject() {
```

Comment:
Yeah... I'm with Christian on this.  Something has, most definitely, changed, for the child.  child.backLinkedField.size(), for instance, is changes.  That is, clearly, a change.

If notifications on backlinks are not supported, that's life.  We need clear documentation of the fact, though.